src/Symfony/Bundle/FrameworkBundle/Resources/config/property_access.xml

```diff
@@ -5,9 +5,29 @@
     xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
     <services>
+
+        <!-- Loader -->
+        <service id="property_access.mapping.chain_loader" class="Symfony\Component\PropertyAccess\Mapping\Loader\LoaderChain" public="false">
+            <argument type="collection" />
+        </service>
+
+        <!-- Class Metadata Factory -->
+        <service id="property_access.mapping.class_metadata_factory" class="Symfony\Component\PropertyAccess\Mapping\Factory\ClassMetadataFactory" public="false">
+            <argument type="service" id="property_access.mapping.chain_loader" />
+            <argument>null</argument>
+        </service>
+
+        <!-- Cache -->
+        <service id="property_access.mapping.cache.apc" class="Doctrine\Common\Cache\ApcCache" public="false">
```

Comment:
@dunglas, could you point me to some leads on how to proceed with this one? All I've found is some sparse WIP documentation about PSR-6 in Symfony